scss/_nav.scss

```diff
@@ -26,18 +26,22 @@
   }
 }
 
-
 //
 // Tabs
 //
 
 .nav-tabs {
   border-bottom: $nav-tabs-border-width solid $nav-tabs-border-color;
 
-  .nav-item {
+  .nav-item,
+  .nav-link {
```

Comment:
We want to handle these 2 markup:
```html
<ul class="nav nav-tabs">
  <li class="nav-item">
    <a class="nav-link" href="#">Item</a>
  </li>
</ul>
```
```html
<nav class="nav nav-tabs">
    <a class="nav-link" href="#">Item</a>
</nav>
```
and the negative `margin-bottom` has to be applied to the direct child of `nav-tabs` in order to hide the `nav-tabs`'s `border-bottom`.  So it has to be applied either on `nav-item` or `nav-link` depending on the markup used.

The only other solution I can see is:
```scss
.nav-tabs {
    .nav-item, .nav-tabs > .nav-link {
        margin-bottom: -$nav-tabs-border-width;
    }
}
```

Then we can get rid of:
```scss
.nav-tabs {
    .nav-item .nav-link {
        margin-bottom: 0;
    }
}
``` 
But as we try to avoid child selector, I opted for the present solution. Let me know what you prefer. Or if there is a better solution that I don't see.